src/status_im2/contexts/chat/home/chat_list_item/view.cljs

```diff
@@ -208,16 +208,40 @@
      {:color color
       :size  :medium}]))
 
+(defn notification
+  [{:keys [muted group-chat unviewed-messages-count unviewed-mentions-count]}]
+  (let [customization-color (rf/sub [:profile/customization-color])
+        unread-messages?    (> unviewed-messages-count 0)
+        unread-mentions?    (> unviewed-mentions-count 0)]
+    [rn/view {:style style/notification-container}
+     (cond
+       muted
+       [icons/icon :i/muted {:color colors/neutral-40}]
+
+       (and group-chat unread-mentions?)
+       [quo/info-count
+        {:style               {:position :relative :right 0}
+         :customization-color customization-color}
+        unviewed-mentions-count]
```

Comment:
the component `quo/info-count` is defining its styles using an `:absolute` positioning by default and setting `:right` as `22`, so I'm overwriting them.

It makes hard to correctly fit this component in any UI.

I think we shouldn't define quo library components's main layout as `:absolute`  (i.e., we can use `:absoulte` but just internally in the component).

wdyt guys?

Reply:
Oh the problem is the default styles are setting `:position :absolute` inside the component, so I'm overwriting them using this `:position :relative`